{
    "description": "Quiero que lleves tus habilidades de manejo de errores en Python a un nivel superior, haciendo que tu código sea más limpio, mantenible y menos propenso a fallos, incluso a medida que crezca en complejidad.\n\nLas estrategias tradicionales de manejo de excepciones en Python, como LBYL (\"Look Before You Leap\") y EAFP (\"Easier to Ask for Forgiveness than Permission\"), funcionan bien en contextos simples, pero rápidamente se vuelven difíciles de manejar a medida que las bases de código crecen. Esto resulta en problemas de legibilidad, complejidad innecesaria y riesgo de errores no manejados adecuadamente, lo cual puede llevar a fallos inesperados.\n\nEn mi charla, \"Más allá de Try-Except: La frontera de Python en el manejo de errores con Monads y Railway Magic\", exploraremos un enfoque innovador para la gestión de errores en Python. Utilizaremos los monads, específicamente con la Results Library, y el método Railway-Driven Development, que nos permitirá adoptar conceptos de programación funcional para construir una estructura sólida y predecible para el manejo de errores.\n\nA través de ejemplos prácticos, demostraré cómo este enfoque facilita la integración de características avanzadas de Python, como pattern matching, haciendo que la gestión de errores sea más clara y eficiente. Al final de la charla, tendrás las herramientas necesarias para implementar un sistema de manejo de errores más robusto y mantenible en tus propios proyectos, superando las limitaciones de las prácticas tradicionales y mejorando significativamente la calidad de tu código.",
    "duration": 3645,
    "language": "eng",
    "recorded": "2025-05-17",
    "related_urls": [
        {
            "label": "Conference Website",
            "url": "https://us.pycon.org/2025/"
        },
        {
            "label": "Presentation Webpage",
            "url": "https://us.pycon.org/2025/schedule/presentation/90/"
        }
    ],
    "speakers": [
        "Sebastian Arias"
    ],
    "thumbnail_url": "https://i.ytimg.com/vi/jTsiY0qtT-M/hqdefault.jpg",
    "title": "Más allá de Try-Except: La frontera de Python en el manejo de errores con Monadas y Railway Magic",
    "videos": [
        {
            "type": "youtube",
            "url": "https://www.youtube.com/watch?v=jTsiY0qtT-M"
        }
    ]
}
